Big-Spender/readme.md

```diff
@@ -112,7 +138,11 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+SELECT sup.*, sum(s.amount) AS supplier_total, s.date FROM suppliers sup
+JOIN spends s ON (sup.id = s.supplier_id)
+WHERE s.date IN ('2021-03-01', '2021-04-01')
+GROUP BY sup.supplier, sup.id, s.date
```

Comment:
I don't think this one called for a grouping by supplier, but it's good practice at grouping